third_party/nvfuser/csrc/expr_simplifier.cpp

```diff
@@ -753,6 +753,17 @@ bool isFlattenedMul(Val* x) {
   return toFlattenedMul(x->definition()) != nullptr;
 }
 
+BinaryOp* toDivModOp(Expr* expr) {
+  if (auto bop = dynamic_cast<BinaryOp*>(expr)) {
```

Comment:
nit: Most of the functions have this pattern of conditional branches where we could reduce indentation levels by negating the condition and exit.

Reply:
changed most of them